cases-SEP24/toml.test.js

```diff
@@ -133,7 +133,13 @@ describe("TOML File", () => {
         "Cannot find an issuer of the enabled currency.",
       ).toBeTruthy();
       json = await server.loadAccount(issuer);
-      expect(url).toEqual(expect.stringContaining(json.home_domain));
+      const expectedDomain = url
```

Comment:
I avoid regex at all costs so I'm going to take you at your word here. It looks like you're extracting the `http://DOMAIN/...` piece correct?

Reply:
I also tend to avoid regex.
However yess this removes the either `http` or `https` and `/...` from `url`(AKA `process.env.DOMAIN`) and `json.home_domain`
such that we only have `DOMAIN` remaining from both to compare.